scaffolder-templates/advanced-workflow/template.yaml

```diff
@@ -162,7 +162,74 @@ spec:
                 - persistencePSQLServiceNamespace
                 - persistencePSQLServicePort
                 - persistencePSQLDatabaseName
+    - title: Create Quay Repository
+      required:
+        - quayCreateNewRepo
+        - CI
+      properties:
+        quayCreateNewRepo:
```

Comment:
I would move all quay related arguments to this "page", like we do for CI/CD (also, rename the title as it will be the quay related page)

Or maybe put `quayCreateNewRepo` in the previous "page" directly so this "page" will be skipped if value set to `No`

Reply:
@gabriel-farache please note the new design:
all quay related information has moved to the Quay 'page'. There, you can select to create a new repository (prompted with all params), use an existing one (prompted for a org and name), or no quay at all (aligns for the case selected no CI method).